test/integration/015_cli_invocation_tests/test_cli_invocation.py

```diff
@@ -111,3 +112,31 @@ def test_postgres_toplevel_dbt_run_with_profile_dir_arg(self):
         # make sure the test runs against `custom_schema`
         for test_result in res:
             self.assertTrue(self.custom_schema, test_result.node.injected_sql)
+
+
+class TestCLIInvocationWithProjectDir(ModelCopyingIntegrationTest):
+
+    @property
+    def schema(self):
+        return "test_cli_invocation_015"
+
+    @property
+    def models(self):
+        return "models"
+
+    @use_profile('postgres')
+    def test_postgres_dbt_commands_with_cwd_as_project_dir(self):
+        self._run_simple_dbt_commands(os.getcwd())
+
+    @use_profile('postgres')
+    def test_postgres_dbt_commands_with_randomdir_as_project_dir(self):
+        workdir = os.getcwd()
+        with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
What!? How did I not know about this context manager? This is awesome!

Reply:
Haha I found out about it today!